RNStoreReview.podspec

```diff
@@ -1,19 +1,37 @@
 require "json"
 version = JSON.parse(File.read("./package.json"))["version"]
 
+folly_compiler_flags = '-DFOLLY_NO_CONFIG -DFOLLY_MOBILE=1 -DFOLLY_USE_LIBCPP=1 -Wno-comma -Wno-shorten-64-to-32'
+
 Pod::Spec.new do |s|
   s.name           = "RNStoreReview"
   s.version        = version
   s.summary        = "App Store Ratings for React Native."
   s.homepage       = "https://github.com/oblador/react-native-store-review"
   s.license        = "MIT"
   s.author         = { "Joel Arvidsson" => "joel@oblador.se" }
-  s.platform       = :ios, "7.0"
+  s.platform       = :ios, "12.4"
   s.source         = { :git => "https://github.com/oblador/react-native-store-review.git", :tag => "v#{s.version}" }
-  s.source_files   = "ios/*.{h,m}"
+  s.source_files   = "ios/*.{h,m,mm}"
   s.preserve_paths = "**/*.js"
+  s.ios.framework  = 'StoreKit'
   s.requires_arc = true
 
   s.dependency "React-Core"
 
+  # This guard prevent to install the dependencies when we run `pod install` in the old architecture.
+  if ENV['RCT_NEW_ARCH_ENABLED'] == '1' then
```

Comment:
FYI Some users enable the New Architecture in their apps on iOS only by setting `:fabric_enabled => true` and as library authors we don't have a way to detect it in Podspec.

Reply:
I know, but afaik there is no way to detect that here, and this is the official way as outlined here: https://github.com/react-native-community/RNNewArchitectureLibraries/tree/feat/back-turbomodule-070#turbomodule-set-up-podspec-file